packages/next/types/webpack.d.ts

```diff
@@ -757,7 +757,7 @@ declare module 'webpack' {
          */
         usedExports?: boolean
         /**
-         *  Recognise the sideEffects flag in package.json or rules to eliminate modules. This depends on optimization.providedExports and optimization.usedExports.
+         *  Recognize the sideEffects flag in package.json or rules to eliminate modules. This depends on optimization.providedExports and optimization.usedExports.
```

Comment:
A few of these seem to be correcting perfectly valid British English spelling to American English (not saying American English is wrong btw)? Is there a convention in the codebase to use one or the other?